ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountStatsMySqlStore.java

```diff
@@ -313,6 +320,18 @@ private void deleteContainerAccountStats(Map<String, StatsSnapshot> prevPartitio
   public StatsWrapper queryAccountStatsByHost(String queryHostname, int port) throws SQLException {
     long startTimeMs = System.currentTimeMillis();
     queryHostname = hostnameHelper.simplifyHostname(queryHostname, port);
+    return queryAccountStatsBySimplifiedHostName(queryHostname, startTimeMs);
+  }
+
+  /**
+   * Query mysql database to get all the container storage usage for given {@code clusterName} and {@code queryHostname} and
+   * construct a {@link StatsSnapshot} from them.
+   * @param queryHostname The simplified hostname to query.
+   * @param startTimeMs The start time to query account stats by host.
+   * @return {@link StatsSnapshot} published by the given simplified hostname.
+   * @throws SQLException
+   */
+  private StatsWrapper queryAccountStatsBySimplifiedHostName(String queryHostname, long startTimeMs) throws SQLException {
```

Comment:
we don't have to pass startTimeMs to this method, we can just call System.currentTimeMillis() to get this value within this method. it's fine to do this for `queryAccountStatsByHost` method since the line in 322 takes no time to finish.

Reply:
Sure, updated.